src/app.js

```diff
@@ -383,6 +384,43 @@ function run () {
   })
   var offsetToLineColumnConverter = new OffsetToLineColumnConverter(compiler.event)
 
+  // ---------------- ContextualListener -----------------------
+  this._components.contextualListener = new ContextualListener({
+    getCursorPosition: () => {
+      return this._components.editor.getCursorPosition()
+    },
+    getCompilationResult: () => {
+      return compiler.lastCompilationResult
+    },
+    highlight: (position, node) => {
+      if (compiler.lastCompilationResult && compiler.lastCompilationResult.source && compiler.lastCompilationResult.source.target === config.get('currentFile')) {
```

Comment:
Please replace `compiler.lastCompilationResult.source.target === config.get('currentFile')` by a check that `position.file` is the file currently shown in the editor.